pages/services/prometheus/0.1.0-2.3.1/managing/index.md

```diff
@@ -0,0 +1,248 @@
+---
```

Comment:
revise this section based on the Prometheus framework.

Reply:
We had referred nifi section, please advise what is the expectation under this